tests/schemaGenerationExample.test.ts

```diff
@@ -0,0 +1,28 @@
+import test from "ava";
+import { generate } from "../src/generate";
+import mockFS from "mock-fs";
+import * as fs from "fs";
+import { getTestDatabase } from "./fixtures/getTestDatabase";
+
+test("simple generation example", async (t) => {
+  const { pool, connectionString } = await getTestDatabase();
+
+  await pool.query(
+    "CREATE TABLE foo (id serial PRIMARY KEY, name text NOT NULL)"
+  );
+
+  mockFS({});
+  await generate({
+    outDir: "/out",
+    db: {
+      connectionString,
+    },
+  });
+  const outFiles = fs.readdirSync("/out/zapatos").sort();
+  t.deepEqual(outFiles, [".eslintrc.json", "schema.d.ts"]);
+
+  const schema = fs.readFileSync("/out/zapatos/schema.d.ts", "utf8");
+
+  t.truthy(schema.includes("export namespace foo {"));
+  t.truthy(schema.includes("export type Table = 'foo'"));
```

Comment:
if it's okay with @jawj , I would also consider using `t.snapshot` instead of checking for specific lines here, that will guard really well against regressions, it will take a full snapshot of the `schema.d.ts` output and store it in a markdown file alongside this test. To update the snapshot you would run `npm run ava ./tests/schemaGenerationExample.test.ts -u`. The snapshot changes would then need to be approved in PR